tests/sections.test.js

```diff
@@ -23,7 +23,7 @@ describe('document should have a valid abstract section', () => {
         type: 'txt',
         data: {
           markers: { abstract: { start: 10 } },
-          content: { abstract: ['This document provides an overview of the system.'] }
+          content: { abstract: ['Abstrat', 'This document provides an overview of the system.'] }
```

Comment:
Is this meant to be Abstract?

Reply:
All of the changes in this file appear unrelated to the stated purpose of the PR?